nsv13/code/modules/custom_shuttle/shuttle_creator_overrides.dm

```diff
@@ -0,0 +1,193 @@
+//=============
+// Overrides shuttle creator to do some WACKY things.
+// Basically custom shuttles no longer exist.
+// Instead the marked turfs get sent to some reserved place and a fighter is created.
+// If you bump the fighter you get teleported to the reserved placed.
+// If you use the shuttle flight computer you get put inside the cockpit of the fighter.
+// Now with 10% less readability.
+//=============
+
+/obj/item/shuttle_creator
+	//The actually fighter object
+	var/obj/structure/overmap/fighter/custom_shuttle/linked_fighter
+	//The output turfs in the reserved land
+	var/list/output_turfs
+
+/obj/item/shuttle_creator/afterattack(atom/target, mob/user, proximity_flag)
+	if(ready && proximity_flag && istype(target, /obj/machinery/computer/custom_shuttle))
+		if(!linked_fighter)
+			to_chat(user, "<span class='warning'>Error, no defined fighter linked to device</span>")
+			return
+		var/obj/machinery/computer/custom_shuttle/console = target
+		console.linked_fighter = linked_fighter
+		linked_fighter.linked_computers += console
+		to_chat(user, "<span class='notice'>Console linked successfully.</span>")
+		return
+	. = ..()
+
+/obj/item/shuttle_creator/shuttle_create_docking_port(atom/target, mob/user)
+
+	if(loggedTurfs.len == 0)
+		to_chat(user, "<span class='warning'>Invalid shuttle (No recorded turfs / No recorded area), restarting bluespace systems...</span>")
+		return FALSE
+
+	//Lets just make it at the first turfs location
+	var/turf/T = loggedTurfs[1]
+	//Calculate width and height of our shuttle bounds
+	//Add 1 so we can have a border.
+	var/list/all_turfs = loggedTurfs
+	//Lets get the boundaries
+	var/min_x = INFINITY
+	var/min_y = INFINITY
+	var/max_x = -INFINITY
+	var/max_y = -INFINITY
+
+	//Do the searching. Doesnt matter too much if its a slow loop, this happens once.
+	for(var/i in 1 to all_turfs.len)
+		var/turf/curT = all_turfs[i]
+		min_x = min(min_x, curT.x)
+		min_y = min(min_y, curT.y)
+		max_x = max(max_x, curT.x)
+		max_y = max(max_y, curT.y)
```

Comment:
```suggestion
	var/min_x
	var/min_y
	var/max_x
	var/max_y

	//Do the searching. Doesnt matter too much if its a slow loop, this happens once.
	for(var/i in 1 to all_turfs.len)
		var/turf/curT = all_turfs[i]
		min_x = min(INFINITY, curT.x)
		min_y = min(INFINITY, curT.y)
		max_x = max(-INFINITY, curT.x)
		max_y = max(-INFINITY, curT.y)
```

Reply:
The suggested changes would just return the last CurT.x and Y in the list rather than the smallest and highest value.